corehq/apps/data_dictionary/util.py

```diff
@@ -398,6 +399,17 @@ def is_case_type_or_prop_name_valid(case_prop_name):
     return match_obj is not None
 
 
+@quickcache(['domain', 'case_type'], timeout=5 * 60)
```

Comment:
Confirming you're intentionally not invalidating the caches for these functions? I think that's fine, with a 5-minute timeout and with this being the data dictionary, just wanted to check you'd considered it.